limesurvey/limesurvey.py

```diff
@@ -284,10 +294,19 @@ def _invoke(self, method: str, *params) -> dict:
             url=limesurvey_api_url, json=payload, timeout=self.timeout
         )
 
-        if response.status_code != requests.status_codes.codes.ok: # pylint: disable=no-member
+        if not response.ok:
             raise Exception(response.text)
 
-        return response.json()
+        json_response = response.json()
```

Comment:
You are still raising an exception

Reply:
Yes, that's better than raising and 500 error

Reply:
Ok, done